include/range/v3/range/access.hpp

```diff
@@ -515,9 +521,8 @@ namespace ranges
         CPP_concept_bool has_member_rend =
             CPP_requires ((T &) t) //
             (
-                _end_::is_sentinel<_rbegin_::_t<CPP_type(T) &>>(t.rend())
-            ) &&
-            std::is_lvalue_reference<T>::value;
+                _end_::is_sentinel<_rbegin_::_t<CPP_type(T) &>>(CPP_fwd(t).rend())
```

Comment:
Ditto "Why are we perfect-forwarding an lvalue?"

Reply:
```suggestion
                _end_::is_sentinel<_rbegin_::_t<CPP_type(T)>>(CPP_fwd(t).rend())
```